cmd/edgectl/aes_install_result.go

```diff
@@ -127,9 +128,6 @@ func (i *Installer) ShowResult(r Result) {
 		}
 
 		if r.Message != "" {
-			i.show.Println()
-			i.show.Println("AES Installation Complete!")
```

Comment:
@brucehorn the one question I have about this change is whether the right messaging is still output for the `KnownLocalClusterResult` case. It would probably be worthwhile to review all of the "successful" exit cases (i.e., those without `err`s to make sure their results are handled properly (based on which fields have been populated in the struct) and the right messages are output. /cc @ark3

Reply:
fixed, see below

Reply:
Thank you!